.expeditor/update_version.sh

```diff
@@ -12,6 +12,7 @@
 set -evx
 
 VERSION=$(cat VERSION)
+ORIGINAL_VERSION=$(git show main:VERSION)
```

Comment:
This needs to be changed for chef 17
`git show chef-17:VERSION`

Reply:
oh crap, I cna't believe I missed that thanks

Reply:
fixed